tests/wrapper/test_transactions_methods.py

```diff
@@ -142,17 +143,9 @@ def test_add_transaction_card_not_present(self):
         self.assertIsInstance(response.instance.source.card, Token)
         self.assertIsInstance(response.instance.source, Source)
 
-    # def test_cancel_transaction(self):
-    # @TODO: terminar cancel transaction
-    #
-    #     self.set_delete_mock(
-    #         200,
-    #         {
-    #             'id': 'foo',
-    #             'resource': 'buyer',
-    #             'deleted': True
-    #         }
-    #     )
-    #
-    #     response = self.client.remove_buyer('foo')
-    #     self.assertEqual(response.status_code, 200, msg=response.data)
+    def test_cancel_transaction(self):
+
+        self.set_delete_mock(200, CancelTransactionCardFactory(id="foo").to_dict())
+
+        response = self.client.remove_buyer("foo")
```

Comment:
Isso deveria ser o método `cancel_transaction`
```suggestion
        response = self.client.cancel_transaction("foo")
```